lmdeploy/cli/utils.py

```diff
@@ -318,13 +318,23 @@ def device(parser):
                                    help='Device type of running')
 
     @staticmethod
-    def meta_instruction(parser):
-        """Add argument meta_instruction to parser."""
+    def chat_template(parser):
+        """Add chat template config to parser."""
 
-        return parser.add_argument('--meta-instruction',
-                                   type=str,
-                                   default=None,
-                                   help='System prompt for ChatTemplateConfig')
+        return parser.add_argument(
+            '--chat-template',
+            type=str,
+            default=None,
+            help='A JSON file that specifies the chat template configuration. '
```

Comment:
It doesn't look like nice very helpful information, too long. Can you use a link url to some user guide?